cranelift/codegen/src/machinst/compile.rs

```diff
@@ -72,20 +72,19 @@ where
                 "Register allocation error for vcode\n{}\nError: {:?}",
                 vcode.show_rru(Some(b.reg_universe())),
                 err
-            );
-            err
-        })
-        .expect("register allocation")
-    };
+            })
+            .expect("register allocation")
+        };
```

Comment:
Something went wrong here. Can you revert this change?

Reply:
It looks like some more went wrong - not just here, during the rebase on top of main, investigating.

Reply:
Re-rebased on `main` this time (didn't catch the branch rename the first time around, but it's all fixed now).